cpu/esp32/esp-wifi/esp_wifi_netdev.c

```diff
@@ -315,17 +311,12 @@ static int _esp_wifi_send(netdev_t *netdev, const iolist_t *iolist)
 
     /* send the the packet to the peer(s) mac address */
     if (esp_wifi_internal_tx(ESP_IF_WIFI_STA, dev->tx_buf, dev->tx_len) == ESP_OK) {
-        #ifdef MODULE_NETSTATS_L2
-        netdev->stats.tx_success++;
-        netdev->stats.tx_bytes += dev->tx_len;
-        #endif
+#ifdef MODULE_NETSTATS_L2
```

Comment:
Wait... This changes behavior. Before this was not in the `#ifdef` block.

Reply:
mhm, right, will remove ... can I amend directly?

Reply:
Yepp